src/git_provider/mod.rs

```diff
@@ -47,11 +48,27 @@ pub trait GitProvider {
     fn set_token(&self, path: &PathBuf) -> Result<String>;
     fn get_token(&self) -> Result<String>;
 
+    fn set_title(&self, branch: &String) -> Result<String> {
+        let title = dialoguer::Input::<String>::with_theme(&ColorfulTheme::default())
+            .with_prompt("Pull request title:")
+            .default(branch.to_owned())
+            .interact()?;
+
+        Ok(title)
+    }
+
+    fn set_body(&self) -> Result<String> {
+        let body = dialoguer::Input::<String>::with_theme(&ColorfulTheme::default())
+            .with_prompt("Pull request body (press enter to leave it empty):")
+            .allow_empty(true)
+            .interact()?;
+        Ok(body)
+    }
```

Comment:
The body should be extracted from the commit body. The question should be: 

```
Set the PR body:
- Paste commit body
- Edit the commit body
```

Edit the commit body, would open an [Editor](https://docs.rs/dialoguer/latest/dialoguer/struct.Editor.html).

Reply:
Good suggestion! I'm on it